mrmap/monitoring/models.py

```diff
@@ -205,10 +244,22 @@ class Meta:
         verbose_name = _('Health state')
         verbose_name_plural = _('Health states')
 
+    @property
+    def resource(self):
+        return self._resource.get_target(self)
+
+    @resource.setter
+    def resource(self, value):
+        self._resource.set_target(self, value)
+
     @property
     def icon(self):
         return Tag(tag='i', attrs={"class": [IconEnum.HEARTBEAT.value]}).render()
 
+    @property
+    def result_view_uri(self):
+        return f"{reverse('monitoring:result_overview')}?monitoring_run={self.monitoring_run_id}&resource={self.resource.id}"
```

Comment:
instead of rewriting reverse function, it would be better to call `return f"{MonitoringResult.get_table_url()}?monitoring_run={self.monitoring_run_id}&resource={self.resource.id}`